### PR TITLE
update the simulator docs [skip ci]

### PR DESCRIPTION
**Pull request by yhwen**
Update the simulator docs.

### CODE REVIEW

File: docs/user_guide/fl_simulator.rst [hunk 1]
```diff
@@ -29,14 +29,16 @@ Command Usage
     optional arguments:
     -h, --help            show this help message and exit
     -w WORKSPACE, --workspace WORKSPACE
-                            WORKSPACE folder
+                        WORKSPACE folder
```
Comment (nvkevlu) [inline docs/user_guide/fl_simulator.rst:32]: These changes to the spacing will cause the final built page to have hard to read spacing.

Comment (nvkevlu) [reply docs/user_guide/fl_simulator.rst:32]: If it is okay with you, I can revise this and then add it as a commit directly to this file.

Comment (yhwen) [reply docs/user_guide/fl_simulator.rst:32]: Please make the changes necessary as we discussed.

**Review by nvkevlu — APPROVED** [review]
Updates made, LGTM

**Comment by yhwen** [comment]
> @yhwen since this affects current user, we need another PR for main branch

Created the PR for main branch. https://github.com/NVIDIA/NVFlare/pull/1187

**Pull request merged**